commerce_coordinator/apps/frontend_app_payment/pipeline.py

```diff
@@ -0,0 +1,24 @@
+"""
+Legacy filter pipeline implementation
+"""
+
+from django.conf import settings
+from openedx_filters import PipelineStep
+from commerce_coordinator.apps.core.constants import PipelineCommand
+from commerce_coordinator.apps.frontend_app_payment.constants import FRONTEND_APP_PAYMENT_CHECKOUT
+
+
+class GetLegacyRedirectUrl(PipelineStep):
+    """
+    Generates legacy redirect url
+    """
+    def run_filter(self, active_order_management_system ,redirect_url):
+
+        if redirect_url is not None or active_order_management_system['active_order_management_system'] != FRONTEND_APP_PAYMENT_CHECKOUT:
```

Comment:
I suppose redirect_url is a dictionary returned from first pipeline step. Isn't that the case?
```suggestion
        if redirect_url.get(redirect_url, None) is not None or active_order_management_system['active_order_management_system'] != FRONTEND_APP_PAYMENT_CHECKOUT:
```